config/envs/default.py

```diff
@@ -127,3 +133,60 @@ class DefaultConfig:
     }
 
     USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "False"))
+
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
+    COF_R2_ROUND_ID = "c603d114-5364-4474-a0c4-c41cbf4d3bbd"
+    COF_R2_SECTION_CONFIG = {
```

Comment:
I think this is too big to be in Config, can we put it in a more context specific class?
Is there a reason this doesn't exist in a store?

Reply:
As part of the conversations discussing the new db schema (reflective of the design and form json changes), it was brought up that the presentation layer (the sections) are a frontend concept (and will also be different in the assessment frontend), the application store holds all the forms associated with an application ID but does not enforce a presentation structure. 
We now configure section/form order in the frontend service using a predefined config json as above. In the longer term it might make more sense to have some mappings from the fund/round store to the application store once we have details on how we want to structure the fund data but this information does not yet exist.

Reply:
OK, can we move this into a related class in that case rather than directly in Config? I don't think this is something we'd want to be overriding on a per environment basis?

Reply:
sure - that's a good suggestion 💯